templates/back-office/transfer-packing-slip.hypr

```diff
@@ -1,5 +1,86 @@
+                    <div>{{ location.address.address1 }}</div>
+                    {% if location.address.address2 %}<div>{{ location.address.address2 }}</div>{% endif %}
+                    {% if location.address.address3 %}<div>{{ location.address.address3 }}</div>{% endif %}
+                    {% if location.address.address4 %}<div>{{ location.address.address4 }}</div>{% endif %}
+                    <div>{{ location.address.cityOrTown }}, {{ location.address.stateOrProvince }} {{ location.address.postalOrZipCode }}</div>
+                    {% if location.phone %}<div>{{ location.phone }}</div>{% endif %}
+                        </td><td>
+                           <div>{{ order.fulfillmentInfo.fulfillmentContact.firstName }} {{ order.fulfillmentInfo.fulfillmentContact.lastNameOrSurname }}</div>
+                    {% if order.fulfillmentInfo.fulfillmentContact.companyOrOrganization %}<div>{{ order.fulfillmentInfo.fulfillmentContact.companyOrOrganization }}</div>{% endif %}
+                    <div>{{ order.fulfillmentInfo.fulfillmentContact.address.address1 }}</div>
+                    {% if order.fulfillmentInfo.fulfillmentContact.address.address2 %}<div>{{ order.fulfillmentInfo.fulfillmentContact.address.address2 }}</div>{% endif %}
+                    {% if order.fulfillmentInfo.fulfillmentContact.address.address3 %}<div>{{ order.fulfillmentInfo.fulfillmentContact.address.address3 }}</div>{% endif %}
+                    {% if order.fulfillmentInfo.fulfillmentContact.address.address4 %}<div>{{ order.fulfillmentInfo.fulfillmentContact.address.address4 }}</div>{% endif %}
+                    <div>{{ order.fulfillmentInfo.fulfillmentContact.address.cityOrTown }}, {{ order.fulfillmentInfo.fulfillmentContact.address.stateOrProvince }} {{ order.fulfillmentInfo.fulfillmentContact.address.postalOrZipCode }}</div>
+                    {% if order.fulfillmentInfo.fulfillmentContact.phoneNumbers.home %}<div>{{ order.fulfillmentInfo.fulfillmentContact.phoneNumbers.home }} (home)</div>{% endif %}
+                    {% if order.fulfillmentInfo.fulfillmentContact.phoneNumbers.work %}<div>{{ order.fulfillmentInfo.fulfillmentContact.phoneNumbers.work }} (work)</div>{% endif %}
+                    {% if order.fulfillmentInfo.fulfillmentContact.phoneNumbers.mobile %}<div>{{ order.fulfillmentInfo.fulfillmentContact.phoneNumbers.mobile }} (mobile)</div>{% endif %}
+                        </td>
+                    </tr>
+                </tbody></table>
+                <table class="grid">
+                    <thead><tr>
+                        <th>Product Code</th>
+                        <th>Product Name</th>
+                        <th>Quantity</th>
+                        <th>Weight</th>
+                    </tr></thead>
+                    <tbody>
+                        {% for packagesItemDetails in packages.packingSlipItemDetails %}
```

Comment:
Please rename `packagesItemDetails` to be singular and remove trailing `s` from `packages`

e.g. `{% for packageItemDetail in package.packingSlipItemDetails %}`